tests/analysis_tests/tests/src/expected/Completion.res.txt

```diff
@@ -1912,26 +1912,6 @@ Path Res
     "tags": [],
     "detail": "module Result",
     "documentation": null
-  }, {
-    "label": "RescriptReactErrorBoundary",
```

Comment:
Do you know what the changes in the completiontests are caused by?

Reply:
I'm not 100% sure. I'm guessing it's because `@rescript/react@npm:0.13.1` is being hoisted.

It can be avoided with some workarounds, but complicated. The easiest way is to connect some vendored library codes by the workspace protocol. We can keep a small part of it for testing purposes.

Reply:
It seems weird that this result is the same even if I manually place it in `node_modules` :thinking:

Reply:
Fixed in https://github.com/rescript-lang/rescript/pull/7357/commits/fcd18ee04a03e56687878b85a0f8dde850207200